doc/DevelopersGuide/scheme_template.F90

```diff
@@ -7,24 +7,23 @@
 ! - scheme must be in its own module (module name = scheme name) and must
 !   have three entry points (subroutines) starting with the name of the module:
 !   module scheme_template -> subroutines scheme_template_{init,finalize,run}
+! 
+! - each .f or .F90 file with CCPP entry point scheme(s) must be accompanied by a 
+!   .meta file containing metadata for the scheme(s) 
```

Comment:
This is another alternative to the current text on line 83 of CompliantPhysicsParams.rst.

Reply:
I used the language you suggested in a previous comment (one or more CCPP entry point schemes).

Reply:
Has Julie's comment been addressed? Same for the `.meta` extension in the following line 12.

Reply:
Now I got it, this is a Fortran file hence no markup. All good.